sdk/appconfiguration/app-configuration/samples/listRevisions.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// This sample demonstrates how to list revisions for a configuration 
+// setting.
+
+// NOTE: replace with import { AppConfigurationClient } from "@azure/app-configuration"
+// in a standalone project
+import { AppConfigurationClient } from "../src"
+
+export async function run() {
+  console.log(`Running listRevisions sample`);
+
+  // You will need to set this environment variable
+  const connectionString = process.env["AZ_CONFIG_CONNECTION"]!;
+  const client = new AppConfigurationClient(connectionString);
+
+  // let's create the setting
+  const originalSetting = await client.addConfigurationSetting({
+    key: `keyWithRevisions-${Date.now()}`,
+    value: "original value"
+  });
+
+  console.log(`First revision created with value ${originalSetting.value}`);
+
+  const newSetting = {
```

Comment:
I feel like people will probably just do `originalSetting.value = 'blah'` and update that. I wonder if you replace originalSetting with just setting and use this pattern if things will be simpler?

Reply:
Agreed, I'll change it.